PIC32/XC32/schemas/PIC32MZ2048EFH100/PIC32MZ2048EFH100_Mikromedia_3_for_PIC32MZ.schema

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
Based on the file name, it will appear as follows:

![image](https://github.com/user-attachments/assets/524cd47a-9e8d-4a91-90aa-466a6e68be18)

Maybe think of a better naming convention, or simply just uppercase it?

Reply:
Ok after consideration, change the `scheme` field in `settings` node to just `MIKROMEDIA` for all files.

Reply:
Changed all 6 files to have the name as "MIKROMEDIA".